backend/fms_core/services/library.py

```diff
@@ -143,3 +144,71 @@ def convert_library(process, platform, sample_source, container_destination, coo
             errors.append(e)
 
     return library_destination, errors, warnings
+
+def set_library_size(sample, library_size = None):
```

Comment:
You can remove the set_library_size function if you make the update library function.